person.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name: 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @age = age
+    @name = name
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    is_of_age? || @parent_permission
+  end
+
+  def of_age?
+    @age >= 18
+  end
```

Comment:
- [ ] You did a great job working on this milestone! 🚀 However, please be careful with the names of your methods, I'm assuming that the rubocop linter suggested changing the method name from `is_of_age` to `of_age` but you forgot to change it too inside your `can_use_services?` method, please do it! 🤓 🔧 
```suggestion
  def can_use_services?
    of_age? || @parent_permission
  end

  def of_age?
    @age >= 18
  end
```